client/payment-details/dispute-details/index.tsx

```diff
@@ -4,9 +4,28 @@
  * External dependencies
  */
 import React from 'react';
+/**
+ * Internal dependencies
+ */
+import { Dispute } from '../../types/disputes';
+import { Card, CardBody } from '@wordpress/components';
+import './style.scss';
+
+interface DisputeDetailsProps {
+	dispute: Dispute;
+}
 
-const DisputeDetails: React.FC< Record< string, never > > = () => {
-	return <></>;
+const DisputeDetails: React.FC< DisputeDetailsProps > = ( { dispute } ) => {
+	return (
+		<div>
+			<Card className="transaction-details-dispute-details-wrapper">
+				<CardBody>
+					<div></div>
+				</CardBody>
+			</Card>
+			<br />
+		</div>
```

Comment:
I mentioned in Slack, we can consider using padding rather than margin here. Something like:

```suggestion
		<div className="transaction-details-dispute-details-wrapper">
			<Card>
				<CardBody>
					<div></div>
				</CardBody>
			</Card>
		</div>
```